types/react-slider/index.d.ts

```diff
@@ -1,36 +1,140 @@
-// Type definitions for react-slider 0.8
+// Type definitions for react-slider 1.0
 // Project: https://github.com/mpowaga/react-slider
 // Definitions by: Jason Unger <https://github.com/jsonunger>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+//                 Wayne Van Son <https://github.com/waynevanson>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/types/react-slider
+// TypeScript Version: 3.6
```

Comment:
What features of typescript 3.6 does this require? Moving from 2.8 to 3.6 means that this update will only be available by default to people on 3.6 or higher. Everybody else will be stuck with the old version.